2016-08-28 22:18:32,057 - Monitor - 124 - ERROR - Cannot connect to mongodb
2016-08-28 22:27:39,273 - tornado.application - 1524 - ERROR - Uncaught exception GET /api/Worker/Monitor/get_alive_workers/ (127.0.0.1)
HTTPServerRequest(protocol='http', host='127.0.0.1:5000', method='GET', uri='/api/Worker/Monitor/get_alive_workers/', version='HTTP/1.1', remote_ip='127.0.0.1', headers={'Connection': 'keep-alive', 'Accept-Encoding': 'gzip, deflate, sdch', 'Accept-Language': 'en-US,en;q=0.8', 'Host': '127.0.0.1:5000', 'User-Agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/52.0.2743.116 Safari/537.36', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8', 'Upgrade-Insecure-Requests': '1'})
Traceback (most recent call last):
  File "/home/emptyset/.pyenv/versions/3.5.1/lib/python3.5/site-packages/tornado/web.py", line 1443, in _execute
    result = method(*self.path_args, **self.path_kwargs)
  File "/home/emptyset/Studio/dHydra-dev/dHydra/web.py", line 62, in get
    self.write( result )
  File "/home/emptyset/.pyenv/versions/3.5.1/lib/python3.5/site-packages/tornado/web.py", line 692, in write
    chunk = escape.json_encode(chunk)
  File "/home/emptyset/.pyenv/versions/3.5.1/lib/python3.5/site-packages/tornado/escape.py", line 88, in json_encode
    return json.dumps(value).replace("</", "<\\/")
  File "/home/emptyset/.pyenv/versions/3.5.1/lib/python3.5/json/__init__.py", line 230, in dumps
    return _default_encoder.encode(obj)
  File "/home/emptyset/.pyenv/versions/3.5.1/lib/python3.5/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/home/emptyset/.pyenv/versions/3.5.1/lib/python3.5/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
TypeError: keys must be a string
2016-08-28 22:27:39,276 - tornado.access - 1946 - ERROR - 500 GET /api/Worker/Monitor/get_alive_workers/ (127.0.0.1) 4.86ms
2016-08-28 22:29:31,751 - tornado.application - 1524 - ERROR - Uncaught exception GET /api/Worker/Monitor/get_alive_workers/ (127.0.0.1)
HTTPServerRequest(protocol='http', host='127.0.0.1:5000', method='GET', uri='/api/Worker/Monitor/get_alive_workers/', version='HTTP/1.1', remote_ip='127.0.0.1', headers={'Host': '127.0.0.1:5000', 'User-Agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/52.0.2743.116 Safari/537.36', 'Cache-Control': 'max-age=0', 'Accept-Encoding': 'gzip, deflate, sdch', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8', 'Accept-Language': 'en-US,en;q=0.8', 'Connection': 'keep-alive', 'Upgrade-Insecure-Requests': '1'})
Traceback (most recent call last):
  File "/home/emptyset/.pyenv/versions/3.5.1/lib/python3.5/site-packages/tornado/web.py", line 1443, in _execute
    result = method(*self.path_args, **self.path_kwargs)
  File "/home/emptyset/Studio/dHydra-dev/dHydra/web.py", line 61, in get
    result = func(self.get_query_arguments)
  File "/home/emptyset/Studio/dHydra-dev/dHydra/Worker/Monitor/Controller.py", line 35, in get_alive_workers
    result = json.dumps(result)
  File "/home/emptyset/.pyenv/versions/3.5.1/lib/python3.5/json/__init__.py", line 230, in dumps
    return _default_encoder.encode(obj)
  File "/home/emptyset/.pyenv/versions/3.5.1/lib/python3.5/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/home/emptyset/.pyenv/versions/3.5.1/lib/python3.5/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
TypeError: keys must be a string
2016-08-28 22:29:31,752 - tornado.access - 1946 - ERROR - 500 GET /api/Worker/Monitor/get_alive_workers/ (127.0.0.1) 2.93ms
2016-08-28 22:31:28,256 - tornado.application - 1524 - ERROR - Uncaught exception GET /api/Worker/Monitor/get_alive_workers/ (127.0.0.1)
HTTPServerRequest(protocol='http', host='127.0.0.1:5000', method='GET', uri='/api/Worker/Monitor/get_alive_workers/', version='HTTP/1.1', remote_ip='127.0.0.1', headers={'Host': '127.0.0.1:5000', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8', 'Accept-Language': 'en-US,en;q=0.8', 'Cache-Control': 'max-age=0', 'Upgrade-Insecure-Requests': '1', 'Connection': 'keep-alive', 'User-Agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/52.0.2743.116 Safari/537.36', 'Accept-Encoding': 'gzip, deflate, sdch'})
Traceback (most recent call last):
  File "/home/emptyset/.pyenv/versions/3.5.1/lib/python3.5/site-packages/tornado/web.py", line 1443, in _execute
    result = method(*self.path_args, **self.path_kwargs)
  File "/home/emptyset/Studio/dHydra-dev/dHydra/web.py", line 61, in get
    result = func(self.get_query_arguments)
  File "/home/emptyset/Studio/dHydra-dev/dHydra/Worker/Monitor/Controller.py", line 35, in get_alive_workers
    result = json.dumps(result, ensure_ascii=False)
  File "/home/emptyset/.pyenv/versions/3.5.1/lib/python3.5/json/__init__.py", line 237, in dumps
    **kw).encode(obj)
  File "/home/emptyset/.pyenv/versions/3.5.1/lib/python3.5/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/home/emptyset/.pyenv/versions/3.5.1/lib/python3.5/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
TypeError: keys must be a string
2016-08-28 22:31:28,257 - tornado.access - 1946 - ERROR - 500 GET /api/Worker/Monitor/get_alive_workers/ (127.0.0.1) 2.93ms
2016-08-28 22:32:28,233 - tornado.application - 1524 - ERROR - Uncaught exception GET /api/Worker/Monitor/get_alive_workers/ (127.0.0.1)
HTTPServerRequest(protocol='http', host='127.0.0.1:5000', method='GET', uri='/api/Worker/Monitor/get_alive_workers/', version='HTTP/1.1', remote_ip='127.0.0.1', headers={'Cache-Control': 'max-age=0', 'Host': '127.0.0.1:5000', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/52.0.2743.116 Safari/537.36', 'Connection': 'keep-alive', 'Upgrade-Insecure-Requests': '1', 'Accept-Language': 'en-US,en;q=0.8', 'Accept-Encoding': 'gzip, deflate, sdch'})
Traceback (most recent call last):
  File "/home/emptyset/.pyenv/versions/3.5.1/lib/python3.5/site-packages/tornado/web.py", line 1443, in _execute
    result = method(*self.path_args, **self.path_kwargs)
  File "/home/emptyset/Studio/dHydra-dev/dHydra/web.py", line 61, in get
    result = func(self.get_query_arguments)
  File "/home/emptyset/Studio/dHydra-dev/dHydra/Worker/Monitor/Controller.py", line 32, in get_alive_workers
    "res"       : json.dumps( alive_workers ),
UnboundLocalError: local variable 'json' referenced before assignment
2016-08-28 22:32:28,235 - tornado.access - 1946 - ERROR - 500 GET /api/Worker/Monitor/get_alive_workers/ (127.0.0.1) 2.98ms
2016-08-28 22:32:45,288 - tornado.application - 1524 - ERROR - Uncaught exception GET /api/Worker/Monitor/get_alive_workers/ (127.0.0.1)
HTTPServerRequest(protocol='http', host='127.0.0.1:5000', method='GET', uri='/api/Worker/Monitor/get_alive_workers/', version='HTTP/1.1', remote_ip='127.0.0.1', headers={'Connection': 'keep-alive', 'Host': '127.0.0.1:5000', 'Cache-Control': 'max-age=0', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8', 'Upgrade-Insecure-Requests': '1', 'Accept-Encoding': 'gzip, deflate, sdch', 'Accept-Language': 'en-US,en;q=0.8', 'User-Agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/52.0.2743.116 Safari/537.36'})
Traceback (most recent call last):
  File "/home/emptyset/.pyenv/versions/3.5.1/lib/python3.5/site-packages/tornado/web.py", line 1443, in _execute
    result = method(*self.path_args, **self.path_kwargs)
  File "/home/emptyset/Studio/dHydra-dev/dHydra/web.py", line 61, in get
    result = func(self.get_query_arguments)
  File "/home/emptyset/Studio/dHydra-dev/dHydra/Worker/Monitor/Controller.py", line 33, in get_alive_workers
    "res"       : json.dumps( alive_workers ),
UnboundLocalError: local variable 'json' referenced before assignment
2016-08-28 22:32:45,290 - tornado.access - 1946 - ERROR - 500 GET /api/Worker/Monitor/get_alive_workers/ (127.0.0.1) 3.28ms
2016-08-28 22:33:12,566 - tornado.application - 1524 - ERROR - Uncaught exception GET /api/Worker/Monitor/get_alive_workers/ (127.0.0.1)
HTTPServerRequest(protocol='http', host='127.0.0.1:5000', method='GET', uri='/api/Worker/Monitor/get_alive_workers/', version='HTTP/1.1', remote_ip='127.0.0.1', headers={'Connection': 'keep-alive', 'Host': '127.0.0.1:5000', 'Accept-Encoding': 'gzip, deflate, sdch', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/52.0.2743.116 Safari/537.36', 'Cache-Control': 'max-age=0', 'Accept-Language': 'en-US,en;q=0.8', 'Upgrade-Insecure-Requests': '1'})
Traceback (most recent call last):
  File "/home/emptyset/.pyenv/versions/3.5.1/lib/python3.5/site-packages/tornado/web.py", line 1443, in _execute
    result = method(*self.path_args, **self.path_kwargs)
  File "/home/emptyset/Studio/dHydra-dev/dHydra/web.py", line 61, in get
    result = func(self.get_query_arguments)
  File "/home/emptyset/Studio/dHydra-dev/dHydra/Worker/Monitor/Controller.py", line 33, in get_alive_workers
    "res"       : json.dumps( alive_workers ),
  File "/home/emptyset/.pyenv/versions/3.5.1/lib/python3.5/json/__init__.py", line 230, in dumps
    return _default_encoder.encode(obj)
  File "/home/emptyset/.pyenv/versions/3.5.1/lib/python3.5/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/home/emptyset/.pyenv/versions/3.5.1/lib/python3.5/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
TypeError: keys must be a string
2016-08-28 22:33:12,567 - tornado.access - 1946 - ERROR - 500 GET /api/Worker/Monitor/get_alive_workers/ (127.0.0.1) 2.96ms
